sqlx-core/src/executor.rs

```diff
@@ -103,7 +103,7 @@ pub trait Executor<'c>: Send + Debug + Sized {
         self.fetch(query).try_collect().boxed()
     }
 
-    /// Execute the query and returns exactly one row.
+    /// Execute the query and return exactly one row.
```

Comment:
I would rephrase this (and other instances of `.fetch_one()`) as 

> Executes the query and returns only the first row, or errors otherwise. 
>
> All subsequent rows returned from the query are discarded.
>
> If the query does not return any rows, `Err(sqlx::Error::RowNotFound)` is returned.